internal/sort/sort.go

```diff
@@ -0,0 +1,28 @@
+package sort
+
+import (
+	"github.com/Prucek/GNU_go_sort/internal/parse"
+	"sort"
+	"strings"
+)
+
+func SortingAlgorithm(lines *[]string, arg *parse.Options) {
```

Comment:
Same as above; when you have to choose a name try to think about it from the caller standpoint. `sort.Lines`, `sort.Slices`, etc. sound better than `sort.SortingAlgorithm`

Reply:
That is true, I am still getting used to working with packages and that totally makes sense